arches/app/etl_modules/bulk_data_deletion.py

```diff
@@ -129,19 +130,32 @@ def get_sample_data(self, nodegroup_id, resourceids):
 
         return sample_data[0:5]
 
-    def delete_resources(self, userid, loadid, graphid, resourceids):
+    def delete_resources(self, userid, loadid, graphid, resourceids, verbose=False):
         result = {"success": False}
-        user = User.objects.get(id=userid)
+        deleted_count = 0
+        user = User.objects.get(id=userid) if userid else None
```

Comment:
Passing None instead of `{}` seems to trigger the delete permissions checks:

https://github.com/archesproject/arches/blob/0c89e7d117f4c639f1ce706f3dfaa82462fb1ca4/arches/app/models/resource.py#L502-L508

Is that desirable here?

Reply:
I'm not sure why we're doing that particular check. I traced it to [this commit](https://github.com/archesproject/arches/commit/2e91ec8a8779732819e79aac65b9d13a27cbe04d) by @chiatt. Maybe it makes sense to set the default value of `user=None` and just check against that? But that's starting to sound like a separate PR.

Reply:
Yes that would sound like a separate PR, so I guess what I'm getting at is unless/until we do that, shouldn't we pass `{}` instead of `None`?

Reply:
@jacobtylerwalls just added #10844 - feel free to add

Reply:
fix in latest